src/com/epam/andrei_eremenko/task_02/two/Calculator.java

```diff
@@ -0,0 +1,65 @@
+package com.epam.andrei_eremenko.task_02.two;
+
+import java.util.*;
+
+public class Calculator {
+    List<String> wholeText;
+    Set<String> croppedText;
+    Iterator<String> iterator;
+    int quantityWords;
+    int occurrences;
+    List<String> repeats;
+
+    StringBuilder stringBuilder;
+
+    String text;
+
+    public Calculator(String text) {
```

Comment:
It will be right to use your composite as a parameter. If you choose a right way to define what composite contain what part of text it will be easy to navigate through it and count anything you want.